.travis.yml

```diff
@@ -8,9 +8,10 @@ env:
 matrix:
     fast_finish: true
     include:
+        - php: 5.6
```

Comment:
I moved this one below in order for Travis to execute the fastest build fist. But actually, it doesn't seem the order has any effect :/